src/components/picker/picker.scss

```
@@ -2,4 +2,5 @@
 
 :host {
     position: relative;
+    display: block;
 }
```

When setting the `display` property of the `:host` selector, that will override the browser styles that hides the element if the attribute `hidden` is set. So we need to add that functionality back:

```suggestion
}

:host([hidden]) {
    display: none;
}
```

---

ah okey, what does that mean exactly? That it is not possible to hide the element with my changes? 🤔

---

No, you can still hide it anyway, but if you write

```html
<p hidden>This should be hidden.</p>
```

that paragraph will indeed be hidden. However, if you have styled it with `display:block` via css, and _then_ add the `hidden` attribute, the css wins. Which perhaps wouldn't normally matter much, but since we're building a component library, that's meant to be used in many circumstances by different people, we want our components to behave like native components (within reason).

---

:zap: